odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -529,19 +529,19 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |web                                         | Done                                            |
 +--------------------------------------------+-------------------------------------------------+
-|web_diagram                                 |                                                 |
+|web_diagram                                 | Nothing to do                                   |
 +--------------------------------------------+-------------------------------------------------+
-|web_editor                                  |                                                 |
+|web_editor                                  | Nothing to do                                   |
 +--------------------------------------------+-------------------------------------------------+
-|web_kanban_gauge                            |                                                 |
+|web_kanban_gauge                            | Nothing to do                                   |
 +--------------------------------------------+-------------------------------------------------+
-| |del| web_planner                          |                                                 |
+| |del| web_planner                          | Nothing to do                                   |
```

Comment:
If this module disappears, you have to see where you have to merge it (web?) for not having this one reported as missing in new version.

Reply:
There are a lot of modules that go to missing in each version because they don't have continuity, it's something difficult to avoid. I think the merging should be done only when it makes sense. And in the web_planner module I think it doesn't makes sense. Odoo just removed it, don't merged it with any other. What I am saying is that, if hypothetically we have a custom module that continues the work of the web_planner, in this case we may not want the merge. So we should do only the essential merges ones.

Reply:
I don't agree. Nobody is going to replace this shitty feature, so we should just ease the migration to this version merging this module with am existing one. You don't decide to install this one in previous version, but was a hard dependency, so you shouldn't need to do any extra stuff of migration.

Reply:
ok